src/armado/compresor.py

```diff
@@ -346,32 +337,26 @@ class ImageManager(BloqueManager):
     def generar_bloques(self, verbose):
         self._prep_archive_dir()
 
-        # pedir todas las imágenes, y ordenarlos en un dict por
-        # su numero de bloque, segun el hash
+        # get all the images, and store them in a dict using its block number, calculated
+        # wiht a hash of the name
         fileNames = []
         for dirname, subdirs, files in os.walk(config.DIR_IMGSLISTAS):
             for f in files:
                 name = os.path.join(dirname, f)[len(config.DIR_IMGSLISTAS) + 1:]
                 fileNames.append(name)
-        if verbose:
-            print("Procesando", len(fileNames), "imágenes")
+        logger.debug("Processing %d images", len(fileNames))
 
         numBloques = len(fileNames) // self.items_per_block + 1
         self.guardarNumBloques(numBloques)
         bloques = {}
         for fileName in fileNames:
             bloqNum = utiles.coherent_hash(fileName.encode('utf8')) % numBloques
             bloques.setdefault(bloqNum, []).append(fileName)
-            if verbose:
-                print("  archs:", bloqNum, repr(fileName))
+            logger.debug("  files:", bloqNum, repr(fileName))
```

Comment:
Esto tira un `TypeError: not all arguments converted during string formatting`,
podríamos poner: `logger.debug("  to block %d: %s", bloqNum, repr(fileName))`